src/test/resources/edu/cornell/kfs/pdp/batch/service/fixture/pdp_ach_test_mixGoodBadLines.csv

```diff
@@ -1,7 +1,7 @@
 EMPL_ID,NET_ID,LAST_NAME,FIRST_NAME,PAYMENT_TYPE,BALANCE_ACCT_,COMPLETE_DT,BANK_NAME,ROUTING_NO,ACCOUNT_NO,ACCOUNT_TYPE
 1122111,laa111,Doe1,John,Directly Deposited,0,01/01/2016,First Bank,109876543,44333222111,Loan
 4455666,jd8888,Doe,Jane,Direct Deposit,1,01/01/2016,Second Bank,666777888,66555444333,Savings
-4455666,jd8888,Doe,Jane,Direct Deposit,1,01/01/2016,First Bank,666777888,6655-444-33,Savings
+4455666,jd8888,Doe,Jane,Direct Deposit,1,01/01/2016,Second Bank,666777888,665-55-444-3--33,Savings

```

Comment:
I see that some of these test values were updated to include dashes, as part of the new testing. Should any of these include spaces as well?